templates/sources.html

```diff
@@ -152,7 +152,11 @@ <h3 id="relationship-name"></h3>
                     }
                 });
                 if(!found){
-                    $('#profile-source-name').append('<option value="'+s.id+'">'+s.name+'</option>');
+                    $('#profile-source-name')
+                        .append($("<option></option>")
```

Comment:
At some point we will have a linter everywhere, but for now I think we're trying to match up with the Airbnb style guide. These double-quoted strings should be single-quoted: https://github.com/airbnb/javascript#strings--quotes

Reply:
Changed my double quotes to singles